src/include/platform/OTARequestorInterface.h

```diff
@@ -191,6 +208,9 @@ class OTARequestorInterface
     // Clear all entries with the specified fabric index in the default OTA provider list
     virtual CHIP_ERROR ClearDefaultOtaProviderList(FabricIndex fabricIndex) = 0;
 
+    using ProviderLocationType = app::Clusters::OtaSoftwareUpdateRequestor::Structs::ProviderLocation::Type;
+    virtual void SetCurrentProviderLocation(ProviderLocationType providerLocation) = 0;
```

Comment:
Same comment about providerLocation being passable as just  <FabricIndex, NodeId, EndpointId> basic types directly

Reply:
I actually have to address https://github.com/project-chip/connectedhomeip/issues/15435 after this PR merges. With that change, I'll be creating a struct to hold the three values you listed here. It could be used here instead of the cluster-object type.

Reply:
Added notes to that issue